examples/00-load/create-platonic-solids.py

```diff
@@ -34,17 +34,14 @@
     'icosahedron',
 ]
 centers = [
-    ( 0, 1, 0),
-    ( 0, 0, 0),
-    ( 0, 2, 0),
+    (0, 1, 0),
+    (0, 0, 0),
+    (0, 2, 0),
     (-1, 0, 0),
     (-1, 2, 0),
 ]
 
-solids = [
-    pv.PlatonicSolid(kind, radius=0.4, center=center)
-    for kind, center in zip(kinds, centers)
-]
+solids = [pv.PlatonicSolid(kind, radius=0.4, center=center) for kind, center in zip(kinds, centers)]
```

Comment:
Some of these lines get pretty long. Is the default line width 100 characters? I saw one nearby with 99. Are we OK with this or should we try something shorter?

Reply:
(Having worked my way all through the changes here I really don't want to touch this setting anymore :laughing:)